modules/programs/alacritty.nix

```diff
@@ -51,6 +50,7 @@ in {
       home.packages = [ cfg.package ];
 
       xdg.configFile."alacritty/alacritty.yml" = mkIf (cfg.settings != { }) {
+        # source = yamlFormat.generate "alacritty.yml" cfg.settings;
```

Comment:
Intentionally commented? If so, then probably good to also add a brief explanation.

Reply:
It would break the configuration of some users because backslashes are replaced:

```
xdg.configFile."alacritty/alacritty.yml".text = replaceStrings [ "\\\\" ] [ "\\" ] (builtins.toJSON cfg.settings);
```